mirage/fixtures/configs.js

```diff
@@ -3,7 +3,6 @@ export default [{
     database: 'mysql',
     enableDeveloperExperiments: true,
     environment: 'development',
-    labs: {subscribers: false},
```

Comment:
Can we keep this line but change it to `labs: {}` instead? Labs itself isn't disappearing, just the subscribers feature.